.github/ISSUE_TEMPLATE/e2e_test_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: e2e test report
+about: Report an e2e test failure.
+title: ''
+labels: ['category: e2e', 'needs triage']
+assignees: ''
+
+---
+
+### Description
+<!-- Before reporting an issue, are you sure this e2e test is failing consistently? -->
+<!-- Add a clear and concise description of what the issue is. Please, be as descriptive as possible. -->
+
+**Error**:
+<!-- The error that the e2e test indicates. -->
+
+**Spec**:
+<!-- The test file that's triggering this error. -->
+
+**Relevant line(s) of code**:
+<!-- Line of code that is triggering this error. -->
+
+**GitHub jobs where this spec is failing**:
+<!-- Can you list all the jobs where the test failing? e.g. "WC - latest | wcpay - shopper", "WC - beta | wcpay - shopper" etc. -->
```

Comment:
This is a bit manual, this would slow me down if I was reporting. Is there a way we can paste this in? 

Worst case (I don't really like this) we could ask the reporter to paste a screenshot of the action checklist.

Another option would be to ask them to mention the first job that fails (i.e. name at least one), and the number of jobs that are failing/affected.

Reply:
Actually, this is not as easy as it sounds. At first, I thought about implementing a list of checkboxes so the reporter could just mark the ones that match the failing jobs. However, we have 16 items and this isn't practical. Furthermore, the jobs may change with time, making this list obsolete.

I don't like the screenshot option because there might be jobs failing in the middle that are failing for different reasons.

I ended up removing this in https://github.com/Automattic/woocommerce-payments/pull/8420/commits/7b0a659931632efa1ffe560d13994b7ec4452a49 since the assignee will see what jobs are failing while fixing it and check that information anyway.